### PR TITLE
Build Help section, community highlights list, external resources, and docs instructions

### PR DESCRIPTION
**Pull request by clay-aptos**
### Description

This also restores the Rust coding guidelines deleted from the repo in:
https://github.com/aptos-labs/aptos-core/commit/6617cc4bb348b42c1da4a59e6946abbdb11dcd03#diff-deb6d1d935e5598636ce61fa0d258708e9daa897885c784892b7cb8d473034fd

### CODE REVIEW

**Comment by clay-aptos** [comment]
@alnoki , I am proposing highlights (docs and code) from community members. See this PR and the staging URL for a preview:
https://deploy-preview-6417--aptos-developer-docs.netlify.app/contribute/community-highlights

**Comment by alnoki** [comment]
> @alnoki , I am proposing highlights (docs and code) from community members. See this PR and the staging URL for a preview: https://deploy-preview-6417--aptos-developer-docs.netlify.app/contribute/community-highlights

@clay-aptos thanks for the mention! The only think that I'll point out is concatenating my GitHub alias name to "alnoki"

**Comment by clay-aptos** [comment]
Let's also link to:
https://stackoverflow.com/questions/tagged/aptos

**Comment by clay-aptos** [comment]
Adding Tau for his guidance as we build this process out:
@rusiqe

**Comment by clay-aptos** [comment]
Hi team,

This is all ready for review except the FAQ, which needs much work internally. I can remove from this PR. What do you think?

File: developer-docs-site/docs/help/aptos-style.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+slug: "aptos-style"
+---
+
+# Follow Aptos Writing Style
+
+When making [site updates](./site-updates.md), Aptos recommends adhering to this writing and formatting style guide for consistency with the rest of Aptos.dev, as well as accessibility directly in GitHub.com and source code editors.
+
+## Hold contributions to high standards
+
+All doc updates should be thorough and tested. This includes external contributions from the community.
```
Comment (clay-aptos) [inline developer-docs-site/docs/help/aptos-style.md]: Test comment.

**Review by clay-aptos — COMMENTED** [review]
Demonstrating editing and reviewing at once

File: developer-docs-site/docs/community/contributions/index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+To add your own contribution here, first submit a new page to Aptos.dev or a product feature in the [`aptos-core`](https://github.com/aptos-labs/aptos-core) repository. Then add a link to the new documentation or code from the table below.
+
+The source materials for both code and docs live in GitHub. See [Contribute](../index.md) for ways to make updates.
+
+| Contribution | Description | Author | Date added/updated |
+| --- | --- | --- | --- |
+| [Multisig Python SDK tooling and tutorial](https://github.com/aptos-labs/aptos-core/pull/6788) | This tutorial introduces assorted [K-of-N multi-signer authentication](../../concepts/accounts.md#multi-signer-authentication) operations | [alnoki](https://github.com/alnoki) | 2023-03-06 |
+| [Remix IDE plugin for Move](https://github.com/aptos-labs/aptos-core/pull/6458) | This tutorial describes a plugin that can develop you develop Aptos Move modules with the Remix IDE. | [0xhsy](https://github.com/0xhsy) | 2023-03-02 |
```
Comment (davidiw) [inline developer-docs-site/docs/community/contributions/index.md:17]: do we want to actively link to PRs?

File: developer-docs-site/docs/community/contributions/index.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+| [How Base Gas Works](../../concepts/base-gas.md) | Aptos transactions by default charge a base gas fee, regardless of market conditions. This document explains that process. | [alnoki](https://github.com/alnoki) | 2022-10-06 |
+
+TODO: Update Multisig and Remix IDE with live documents once in place. Not Multisig is landing in two parts:
+https://github.com/aptos-labs/aptos-core/pull/6788/
+https://github.com/aptos-labs/aptos-core/pull/6356
```
Comment (davidiw) [inline developer-docs-site/docs/community/contributions/index.md:22]: This feels a bit out of place for a docs site.

File: developer-docs-site/docs/community/site-updates.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+## tl;dr
+
+Simply click **Edit this page** at the bottom of any location to go to the source and trigger editing there. The contents are in [Markdown](https://www.markdownguide.org/basic-syntax/) format. You may then edit in browser and use the *Preview* function to view your changes.
+
+Here are the basic steps for editing in your web browser:
+
+1. Click **Edit this page** at the bottom to get started.
+2. Modify and add source Markdown files in the [developer-docs-site](https://github.com/aptos-labs/aptos-core/tree/main/developer-docs-site) directory.
+3. See your changes in Netlify (by swapping `prnumber` in):
```
Comment (davidiw) [inline developer-docs-site/docs/community/site-updates.md:20]: I don't think this is available to the general public, so they should verify locally or we can make this instruction for reviewers.

File: developer-docs-site/docs/community/site-updates.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+       * To add a new page, navigate to the relevant subdirectory of the [developer-docs-site/docs/](https://github.com/aptos-labs/aptos-core/tree/main/developer-docs-site/docs/) directory, click **Add file**, give it a name, append the `.md` file extension, include your contents, and create a pull request to have it reviewed:
+       ![v-fn-network.svg](../../static/img/docs/add-file-in-GH.png)
+    2. Via local editor - for more complex, multi-page changes, use your preferred source code editor to navigate to and update the source Markdown files in GitHub. See our [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) README for `git clone` instructions.
+2. For web edits, use the *Preview* function at top to see your updates in browser.
+3. For local edits, use the [local doc build instructions](https://github.com/aptos-labs/aptos-core/blob/main/developer-docs-site/README.md) to see your updates at: [http://localhost:3000](http://localhost:3000)
+4. After creating the pull request, use the *Deploy Preview* in Netlify to see your updates made in web browser or via local editor by replacing the *prnumber* with your own in:
```
Comment (davidiw) [inline developer-docs-site/docs/community/site-updates.md:57]: repeat feedback from above

File: developer-docs-site/docs/community/site-updates.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+[https://deploy-preview-prnumber--aptos-developer-docs.netlify.app/](https://deploy-preview-prnumber--aptos-developer-docs.netlify.app/)
+5. Have at least two members of Aptos Labs review and test your changes.
```
Comment (davidiw) [inline developer-docs-site/docs/community/site-updates.md:59]: verified reviewers

**Review by davidiw — COMMENTED** [review]
Please adjust, looks good otherwise.

File: developer-docs-site/docs/community/site-updates.md (hunk 4 above)
Comment (clay-aptos) [reply developer-docs-site/docs/community/site-updates.md:20]: I tested in Incognito. For better or worse, it is available. I mention the ability to verify locally elsewhere.

File: developer-docs-site/docs/community/contributions/index.md (hunk 2 above)
Comment (clay-aptos) [reply developer-docs-site/docs/community/contributions/index.md:17]: No, I was hoping at least one would land by now. :-)

I have moved this and the TODO that follows to the issue:
https://github.com/aptos-labs/aptos-core/issues/6311

File: developer-docs-site/docs/community/contributions/index.md (hunk 3 above)
Comment (clay-aptos) [reply developer-docs-site/docs/community/contributions/index.md:22]: Moved to the issue:
https://github.com/aptos-labs/aptos-core/issues/6311

File: developer-docs-site/docs/community/site-updates.md (hunk 6 above)
Comment (clay-aptos) [reply developer-docs-site/docs/community/site-updates.md:59]: Changed to "two verified reviewers of Aptos Labs" here. Is that what you meant?

File: developer-docs-site/docs/community/site-updates.md (hunk 5 above)
Comment (davidiw) [reply developer-docs-site/docs/community/site-updates.md:57]: the comment about netlify. all good

**Review by davidiw — APPROVED** [review]
awesome

**Pull request merged**